lab-michael/gulpfile.js

```diff
@@ -0,0 +1,24 @@
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
+
+
+gulp.task('welcome', function() {
+  console.log('Welcome and Good Day');
+});
+
+gulp.task('tester', function() {
+  gulp.src('./test/test-*.js')
+  .pipe(mocha());
+});
+
+gulp.task('linter', function() {
+  gulp.src('./**/*.js')
+  .pipe(eslint());
```

Comment:
You're missing a couple of pipes here for the linter. Go read up on the gulp-eslint docs to see if you can find it... they're right in the 'getting started' section of the docs.